src/test/resources/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-SymbolicName: nuxeo-groups-rights-audit-tests
```

Comment:
The convention we use for bundle symbolic names is of the form `org.nuxeo.ecm.platform.groups.audit.tests` (similar to Java package, can be simplified a bit if needed). Please also change the `Bundle-SymbolicName` of the non-test `MANIFEST.MF`.